.github/workflows/continuous-integration-workflow.yml

```diff
@@ -83,3 +85,29 @@ jobs:
           file: ${{ matrix.config.dockerfile }}
           build-args: ${{ steps.build_args.outputs.args }}
           push: true
+
+  merge-docker-manifest:
+    runs-on: ubuntu-latest
+    needs: CI
+    strategy:
+        matrix:
+          config:
+            - {dockerfile: 'fedora',   tag: 'latest'}
+            - {dockerfile: 'ubuntu',   tag: 'latest'}
+    env:
+      docker-tag: ghcr.io/kokkos/ci-containers/${{ matrix.config.dockerfile }}:${{ matrix.config.tag }}
+    steps:
+      - name: Login to Github Container Registry
+        if: github.repository_owner == 'kokkos' && ( github.event_name == 'push' ||  github.event_name == 'schedule' )
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create and push manifest images to Github Container Registry
```

Comment:
This part is not a 100% needed, but it is nice to have a `kokkos/ci-containers/fedora:latest` tag that is multiarch.